pkg/api/api.proto

```diff
@@ -231,6 +235,31 @@ message StateChangeEvent {
   Container container = 3;
 }
 
+message ValidateContainerAdjustmentRequest {
+    // Pod of container being adjusted.
+    PodSandbox pod = 1;
+    // Container being adjusted in its pristine state.
+    Container container = 2;
+    // Pending container adjustments.
+    ContainerAdjustment adjust = 3;
+    // Pending updates to other containers.
+    repeated ContainerUpdate update = 4;
+    // Plugins that made the adjustments and updates.
+    OwningPlugins owners = 5;
+    // Plugins consulted for adjustments and updates.
+    repeated PluginInstance plugins = 6;
```

Comment:
Because this is a `repeated` field, should this be `plugin`?

Reply:
When translated to golang, that would become a `Plugin []*PluginInstance`, whereas now we have `Plugins []*PluginInstance` which I think is better.

But I think I know where your comment is stemming from, probably looking at `update` in the same message which is a slice too, but named in singular. We have been (grammatically) a bit inconsistent, though consistently inconsistent at that, with naming repeated fields. Container updates/evictions, although repeated/slices, are always named in singular. All other repeated fields, except `env`  (for "environment") and the individual hook slices in `Hooks` (which follow the naming of the corresponding OCI Spec fields) are in plural.